### ISSUE TITLE
Example uses deprecated Grizzly functionality

### ISSUE THREAD

**Issue by mikaello** [joanlopez/grafonnet-example#1]
Thanks for your example!

Just wanted to raise the issue that the example is using deprecated syntax for defining the dashboards:

https://github.com/joanlopez/grafonnet-example/blob/c68320c140bd20e9e590a549c22649ec5138ed44/dashboards.libsonnet#L2-L4

Ref Grizzly documentation regarding [Hidden Elements](https://grafana.github.io/grizzly/hidden-elements/):

> **NOTE**: This functionality is deprecated. It was necessary to support the formats within [Monitoring Mixins](https://github.com/monitoring-mixins/docs). However, using explicit resources provides a cleaner and language agnostic way to describe resources and their metadata. If you have existing libaries that use hidden elements, it isn't too hard to write small functions to wrap these into resources.

I think your example dashboard instead should be something like this:

```jsonnet
local grr = import 'github.com/grafana/jsonnet-libs/grizzly/grizzly.libsonnet';

{
  dashboards: [
    grr.dashboard.new('example', (import 'example.jsonnet')),
  ],
}
```

**Comment by joanlopez** [comment]
Hey @mikaello,

Thanks for reporting it! Would you mind opening a PR? Thanks!

**Comment by mikaello** [comment]
Sure: https://github.com/joanlopez/grafonnet-example/pull/2

**Comment by joanlopez** [comment]
Closed by #2, thanks so much @mikaello! 🙏🏻 🙏🏻
